tests/draft-next/optional/format/time.json

```
@@ -205,6 +205,11 @@
                 "description": "offset not starting with plus or minus",
                 "data": "08:30:06#00:20",
                 "valid": false
+            },
+            {
+                "description": "contains alphabets",
```

```suggestion
                "description": "contains letters",
```

And in others below.

(alphabet is generally used for the entire a through z at once; "letters" or "alphabetical characters" if you've got just a few of them)